api/test/h3-data/h3-risk-map.spec.ts

```diff
@@ -344,6 +509,13 @@ describe('H3 Data Module (e2e) - Risk map', () => {
       fakeColumn,
     });
 
+    const indicatorH3Data = await h3DataMock(
```

Comment:
you can remove the `const indicatorH3Data = ` part, as it's never referenced again (your IDE should probably pick this up for you)